apps/vanilla/src/pages/index.css

```
@@ -2,4 +2,90 @@
 
 .title {
   color: var(--heading-color-1);
+  font-size: 4rem;
+  font-family: Arial, Helvetica, sans-serif;
+  letter-spacing: 0.5rem;
+  margin-bottom: 40px;
+}
+
+.select {
+  margin: 20px 0;
+}
+
+.table__row td {
+  padding: 10px;
+}
+
+.table th td {
+  border: var(--border-color) 1px solid;
+  width: 100%;
+}
+
+.table__row {
+  background-color: var(--background-color);
+}
+
+.table__row:nth-child(even) {
+  background-color: var(--dark-background-color);
+  color: var(--primary-font-color);
+}
+
+.table__head {
+  border: 1px solid var(--border-color);
+  font-size: 1.3rem;
+  background-color: var(--primary-color);
+  color: var(--primary-contrast-font-color);
+}
+
+.table__head th {
+  padding: 20px;
+  text-align: start;
+}
+
+.table__row-image {
+  max-width: 200px;
+  max-height: 200px;
+}
+
+.pagination {
+  display: flex;
+  width: 60%;
+  justify-content: space-between;
+}
+
+.pagination__numeric {
+  display: flex;
+  width: 100%;
+  justify-content: space-around;
+}
+
+.pagination__button {
+  background-color: var(--primary-color);
+  color: var(--primary-contrast-font-color);
+  margin: 20px 0;
+  padding: 5px 15px;
+  border-radius: 10%;
+  cursor: pointer;
+}
+
+.pagination__button.active {
```

Please stick to [BEM](https://wiki.saritasa.rocks/frontend/languages/css#bem)
```suggestion
.pagination__button_active
```